client/makelunch.js

```diff
@@ -130,6 +130,10 @@ Template.registerHelper('todaysISODate', function () {
 })
 
 Template.card.events({
+  'click .photoFrame': function (evt, tpl) {
+    $('.card').removeClass('visible')
+    tpl.$('.card').addClass('visible')
```

Comment:
The first one removes it everywhere, the second one adds it only to the template you've clicked.